src/backend/aspen/database/models/sample.py

```diff
@@ -183,6 +184,11 @@ class Sample(idbase, DictMixin):  # type: ignore
     )
 
     # location
+    location_id = Column(
```

Comment:
Can we set up `collection_location` as a relationship with the `locations` table so we don't wind up conflicting with the current `location` char field?

Reply:
Yeah, I can do that